index.js

```diff
@@ -1,11 +1,18 @@
 const { constants, hiscores } = require("osrs-api");
 const express = require('express')
+const path = require('path')
 const app = express()
-
 const port = 3000
 
 
-app.get('/:username', function(req, res, error){
+
+app.use(express.static(path.join(__dirname, 'public')));
+
+app.get('/', (req, res) => {
+  res.sendFile(path.join(__dirname + 'index.html'))
+});
+
+app.get('/search/:username', function(req, res, error){
```

Comment:
You might want to update the readme to reflect this

Reply:
Does the stuff in `index.html` show up when you hit this route?

Reply:
yes it does

Reply:
Then I understand less than I thought I did

Reply:
No sorry I was mistaken. When you hit the :username route, the index.html does NOT show up. It only shows up when you hit the '/' root.